fuel-client/Cargo.toml

```diff
@@ -17,14 +17,15 @@ path = "src/main.rs"
 [dependencies]
 anyhow = "1.0"
 clap = { version = "3.1", features = ["derive"] }
-cynic = { version = "1.0", features = ["surf"] }
+cynic = { version = "2", features = ["http-surf"] }
```

Comment:
@Dentosal 's comments on the initial issue mentioned we might want to wait a week or 2 for initial bugfixes, did you encounter any weirdness when migrating to the new version?

Also any reason for the downgrade in version precision from x.x to x?

Reply:
"1.0" and "1" are the same=)
<img width="870" alt="image" src="https://user-images.githubusercontent.com/18346821/203793243-55766b77-a3a2-44fb-b0c8-2a52399a59b6.png">